### PR TITLE
QBO Bank Feeds - document new functionality - 1

### PR DESCRIPTION
**Pull request by d-laing**
# Description

Updates the QuickBooks Online Bank Feeds documentation with several new pieces of functionality.

1. Support for historic bank feeds up to seven days old.
2. Bank accounts which are disconnected by SMB users in QBO are set to `disconnected` in Codat, if not queried for 10 days.
3. French language support in the QBO Bank Feeds Link UI and in the optional custom text call-outs.
4. The `linkUrl` contains a one-time password `otp` that expires after one hour.
5. An SMB user can now revoke their one-time credentials in the Link UI.
6. Source bank accounts can now have a `disconnected` status.
7. Pushing bank transactions to QBO: explain `clearedOnDate` more fully; explain historic transactions; show which fields are supported/required/not supported in the request body
8. Removed the content about customising the Link flow by changing the colours and branding - the SMB user won't see the standard Link flow with this integration.

Also makes several general enhancements, including:
-  Rewriting the introduction to hopefully explain the solution more clearly.
- Divides the content on the SMB user connection flow into two, to avoid a 14 step procedure.
- Various other improvements to terminology, including:
  - "source bank account" and "target bank account" 
  - the QBO Bank Feeds Link UI, rather than _Link site_ 

## Type of change

Please delete options that are not relevant.

- [x] New document(s)/updating existing
- [x] Fixes

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-push-bank-transactions.md [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
 
-Because of the way bank transactions work, we recommend you post seven days of transactions on the initial push. For subsequent pushes, we recommend you post daily transaction data.
+QBO requires that bank transactions are sent from Codat in chronological order. Therefore, we recommend you post seven days of historic transactions on the initial push. For subsequent pushes, we recommend you post daily transaction data, which will be sent to QBO on a daily schedule.
 
 ## Push bank transactions to QuickBooks Online
 
 Make the following requests to the Codat API. All push requests are asynchronous.
 
-1. Push bank transactions from an SMB user's connected bank account using the <a className="external" href="/accounting-api#/operations/post-bank-transactions" target="_blank">Create bank transactions</a> endpoint. Note that:
+1. Push bank transactions from an SMB user's connected bank account using the [Create bank transactions](/accounting-api#/operations/create-bank-transactions) endpoint. Note that:
```
Comment (pzaichkina) [inline docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-push-bank-transactions.md:34]: not specific to your changes, but is there value in cleaning up a bit of formatting here?
The first code snippet sort of comes out of nowhere and has no spacing, and the title of the second one gets lost on the page. Same with the statement right underneath it. 
For the title, you can just do ```json title="Example request body (all fields are mandatory)"

image

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds.md [hunk 2]
```diff
@@ -3,58 +3,65 @@ title: "QuickBooks Online Bank Feeds"
...
-Bank transactions are automatically pushed from Codat to QBO on a daily basis. To view the latest bank transactions, SMB users can click **Update** in the Transactions area in QBO. This triggers an immediate push of bank transactions and overrides the daily schedule.
+Pushed bank transactions are automatically sent from Codat to QBO on a daily schedule. The SMB user can click **Update** in the **Transactions** area in QBO to override the daily schedule and view their latest transactions immediately.
 
-### Disabling bank feeds
+### Disconnecting bank feeds
 
-If an SMB customer turns off a bank feed for a connected account, you won't be able to push further bank transactions to that account.
+The SMB user can disable bank feeds for their target accounts using the options in QBO. This stops a target account from receiving any further transactions, which means that any subsequent pushes will fail.
+
+When a bank feed is disabled in QBO, it takes 10 days for the status of the connected bank account in Codat to change to `disconnected`.
```
Comment (Huweey) [inline docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds.md:48]: The DataConnection of a company will also be set to "unlinked" if all the bankAccounts have statuses of "disconnected". Like the BankAccount Status, it won't update immediately and could take up to a day to unlink the connection.

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-smb-user.md [hunk 3]
```diff
@@ -1,44 +1,70 @@
...
-1. They select the **Intuit QuickBooks Bank Feeds** tile, then click **Next**.
+## Summary of SMB user connection flow
+
+To connect their bank accounts to QBO, the SMB user performs the following steps in the QBO Bank Feeds Link UI and in their QBO account.
+
+### Generate one-time credentials
+
+1. From within your application, the SMB user opens the `linkUrl` in their browser.
+
+2. They select the **Intuit QuickBooks Bank Feeds** tile, then click **Next**.
```
[the remark this answers is not in the document]
Comment (d-laing) [reply docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-smb-user.md:23]: Verified. I've removed steps 2 and 3.

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds.md (hunk 2 above)
Comment (d-laing) [reply docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds.md:48]: Very useful information.

I've added your text and edited the whole section to hopefully make it clearer.

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-push-bank-transactions.md (hunk 1 above)
Comment (d-laing) [reply docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-push-bank-transactions.md:40]: Yes, I think this is worth cleaning up. I didn't think to add titles to the code snippets.

Comment (d-laing) [reply docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-push-bank-transactions.md]: I've moved the bullet points out of the procedure and into the introduction. Hopefully, that helps with the request seeming to come out of nowhere.
Added a title for the request.
I decided to leave the statement at the end ("The balance of the last bank transaction...") in the same place; it seems preferable to give this information in the context of the task. Hopefully it won't get lost now.

**Review by Huweey — APPROVED** [review]

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md [hunk 4]
```diff
@@ -2,65 +2,74 @@
 title: "Set up the QuickBooks Online Bank Feeds integration"
 description: "Set up our integration with QuickBooks Online Bank Feeds."
 sidebar_label: Setting up
-createdAt: "2022-09-05T13:13:20.253Z"
-updatedAt: "2023-01-11T15:07:17.188Z"
 ---
 
-Set up the QuickBooks Online Bank Feeds integration to let your SMB users connect bank accounts from your application to QuickBooks Online (QBO).
+Set up the QuickBooks Online Bank Feeds integration to let your SMB customers connect source bank accounts in your application to target accounts QuickBooks Online (QBO).
```
Comment (pzaichkina) [inline docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md:7]:
```suggestion
Set up the QuickBooks Online Bank Feeds integration to let your SMB customers connect source bank accounts in your application to target accounts in QuickBooks Online (QBO).
```

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md [hunk 5]
```diff
@@ -2,65 +2,74 @@
...
-- You've customized Link for your institution.
+- Your Solutions Engineer has provided you with access to the QuickBooks Online Bank Feeds integration.
+- SMB users who will connect to QBO Bank Feeds are located in the US and Canada only.
+- In the Codat Portal, you've enabled [Bank accounts](/accounting-api#/schemas/BankAccount) and [Bank transactions](/accounting-api#/schemas/BankTransactions) in your [Data type settings](/core-concepts/data-type-settings).
+- You've customized the QBO Bank Feeds Link UI for your institution.
 
 ## Enable the QBO Bank Feeds integration
 
-1. Go to the <a className="external" href="https://app.codat.io/settings/integrations/bankfeeds" target="_blank">**Bank feed integrations**</a> page in the Codat Portal.
+1. Go to the [**Bank feed integrations**](https://app.codat.io/settings/integrations/bankfeeds) page in the Codat Portal.
```
Comment (pzaichkina) [inline docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md:20]: do we not want to say Settings > Integrations > Bank feeds here?

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md [hunk 6]
```diff
@@ -2,65 +2,74 @@
...
+
+:::
+
 ## Add a custom callout to the Link Site
 
-You can add a custom text callout to the QBO Bank Feeds Link site to provide additional guidance to users who are connecting their bank accounts to QBO.
+You can add a custom text callout, in both French and English, to the QBO Bank Feeds Link UI. These can provide additional guidance to your SMB users on connecting their bank accounts to QBO.
 
-1. Go to the <a className="external" href="https://app.codat.io/settings/integrations/bankfeeds" target="_blank">**Bank feed integrations**</a> page in the Codat Portal.
+1. Go to the [**Bank feed integrations**](https://app.codat.io/settings/integrations/bankfeeds) page in the Codat Portal.
```
Comment (pzaichkina) [inline docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md:34]: same question here

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md [hunk 7]
```diff
@@ -122,11 +131,17 @@ See the next procedure for details on the functionality to provide.
 
    The endpoint returns a `200` response and the list of created bank accounts.
 
-4. Redirect the SMB user to the `linkUrl` returned in the response from the `POST /connections` endpoint (see step 2).
+4. Redirect the SMB user to the `linkUrl` returned in the response from the Create a data connection endpoint (see step 2).
```
Comment (pzaichkina) [inline docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md:134]: is there a need to link to the Create a data connection endpoint?

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md (hunk 5 above)
Comment (d-laing) [reply docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md:20]: Yes, maybe we should go back to using menu paths. However, I don't think menu paths that are also links look very good visually.

Comment (d-laing) [reply docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md]: Changed to Settings > Integrations > Bank feeds as well as the direct link.

File: docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md (hunk 7 above)
Comment (d-laing) [reply docs/bank-feeds-api/qbo-bank-feeds/qbo-bank-feeds-setup.md]: Added link.

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**